ports/xapian/vcpkg.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "xapian",
+  "version": "1.4.21",
+  "description": "Xapian is an Open Source Search Engine Library, released under the GPL v2+. It's written in C++, with bindings to allow use from Perl, Python 2, Python 3, PHP 5, PHP 7, Java, Tcl, C#, Ruby, Lua, Erlang, Node.js and R (so far!)",
+  "homepage": "https://github.com/xapian/xapian",
+  "license": "GPL-2.0-or-later",
+  "supports": "!(arm | arm64 | uwp)",
```

```suggestion
  "supports": "!(arm | uwp)",
```
`arm` include `arm64`.